Java/parkservices/src/main/java/com/amazon/randomcutforest/parkservices/ThresholdedRandomCutForest.java

```diff
@@ -109,7 +109,7 @@ public ThresholdedRandomCutForest(Builder<?> builder) {
         forest = builder.buildForest();
         preprocessorBuilder.weights(builder.weights);
         preprocessorBuilder.weightTime(builder.weightTime.orElse(1.0));
-        preprocessorBuilder.timeDecay(forest.getTimeDecay());
+        preprocessorBuilder.timeDecay(builder.transformDecay.orElse(forest.getTimeDecay()));
```

Comment:
What is transform decay? How is it different from time decay?

Reply:
TransformDecay is the exponential moving average during normalization -- this value is subtracted from the (corresponding element of) the input value and divided by the corresponding deviation (with the same exponential weights). This corresponds to the 50-value moving average or 100-value moving average computations; the transform decays would be 0.02 or 0.01. 

TimeDecay (existed earlier in the library) corresponds to the half-life of detain the weighted online sampling. 

It may make sense to use the same number -- but it is a flexibility to be able to use two different numbers. Anomaly detection applications are not particularly sensitive to the transformDecay and hence it did not have to exist earlier. Many folks believe that the moving average computation is of interest to forecasting, specially long term trends. So making that available as an option makes more sense. The example RCFCasterExample.lava shows how to use it (and has some comments). It makes some difference in how quickly does forecasts adapt to changes -- but does not have a strong impact for stationary sequences.